class ConnectionError(Exception):
    pass

class CheckoutException(Exception):
    pass
